src/gui/issueswidget.cpp

```diff
@@ -200,4 +261,72 @@ void IssuesWidget::slotItemContextMenu()
     ProtocolWidget::showContextMenu(this, _model, rows);
```

Comment:
I can't comment line 259 but its still using the sort model instead of the filter model.
Still crashing when right clicking an issue.
```
09-28 13:32:31:040 [ warning default ]:	QSortFilterProxyModel: index from wrong model passed to mapToSource
09-28 13:32:31:040 [ fatal default ]:	ASSERT: "!"QSortFilterProxyModel: index from wrong model passed to mapToSource"" in file C:\_\17a9f6ae\qtbase-everywhere-src-5.12.10\src\corelib\itemmodels\qsortfilterproxymodel.cpp, line 548